Model/Sales/TrackTraceHolder.php

```diff
@@ -296,19 +296,17 @@ private function convertDataForCdCountry($magentoTrack)
         }
 
         $products = $this->getItemsCollectionByShipmentId($magentoTrack->getShipment()->getId());
-
         foreach ($products as $product) {
             $myParcelProduct = (new MyParcelCustomsItem())
                 ->setDescription($product['name'])
                 ->setAmount($product['qty'])
                 ->setWeight($this->getWeightTypeOfOption($product['weight']))
                 ->setItemValue($product['price'] * 100)
-                ->setClassification((int) $this->hsCode('catalog_product_entity_int', $product['product_id'], 'classification'))
-                ->setCountry('NL');
+                ->setClassification((int) $this->getAttributeValue('catalog_product_entity_int', $product['product_id'], 'classification'))
+                ->setCountry((string) $this->getCountryOfOrigin($product));
```

Comment:
Waarom wordt dit gecast?

Reply:
De reden dat dit gedaan is, is omdat de data die je krijgt van $this->getCountryOfOrigin($product) waarde is die direct uit de database gehaald is.